source/_components/sensor.elta.markdown

```diff
@@ -0,0 +1,40 @@
+---
+layout: page
+title: ELTA Sensor
+description: "Instructions on how to set up Elta sensors within Home Assistant."
+date: 2017-02-05 08:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: http://www.elta.gr/Portals/0/elta.jpg
+ha_category: Sensor
+ha_release: 0.36
```

Comment:
Needs to be updated to the next release. I think i would recommend 0.39 since this most likely won't be merged in by next weekend,